modules/test/playwright/tests/site-admin-web/main/mySites.spec.ts

```diff
@@ -0,0 +1,553 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
+import {isolatedSiteTest} from '../../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../../fixtures/loginTest';
+import {pageEditorPagesTest} from '../../../fixtures/pageEditorPagesTest';
+import {pagesAdminPagesTest} from '../../../fixtures/pagesAdminPagesTest';
+import {siteSettingsPagesTest} from '../../../fixtures/siteSettingsPagesTest';
+import {usersAndOrganizationsPagesTest} from '../../../fixtures/usersAndOrganizationsPagesTest';
+import getRandomString from '../../../utils/getRandomString';
+import {
+	performLoginViaApi,
+	performLogout,
+	userData,
+} from '../../../utils/performLogin';
+import {waitForAlert} from '../../../utils/waitForAlert';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	loginTest(),
+	isolatedSiteTest,
+	pageEditorPagesTest,
+	pagesAdminPagesTest,
+	siteSettingsPagesTest,
+	usersAndOrganizationsPagesTest
+);
+
+let layout;
+
+test.beforeEach(async ({apiHelpers, pageEditorPage, pagesAdminPage, site}) => {
+	await pagesAdminPage.goto(site.friendlyUrlPath);
+
+	layout = await apiHelpers.jsonWebServicesLayout.addLayout({
+		groupId: site.id,
+		options: {type: 'content'},
+		title: 'Site Page',
+	});
+
+	await pageEditorPage.goto(layout, site.friendlyUrlPath);
+
+	await pageEditorPage.addWidget('Community', 'My Sites');
+
+	await pageEditorPage.publishPage();
+});
+
+test(
+	'Assert a user can navigate to a Site public page via MySites widget',
+	{
+		tag: '@LPD-66998',
```

Comment:
Since the tags are pointing to the migration ticket instead of a bug fix ticket let's remove all of them.